src/pds/registrysweepers/ancestry/__init__.py

```diff
@@ -0,0 +1,104 @@
+import logging
```

Comment:
I would have defined the subpackages as: `pds/registry/sweepers/ancestry` since registry is the parent package of registry-sweeper. But that could be discussed. Not a big deal anyway.

Reply:
@tloubrieu-jpl do you mean there's an existing/published package `pds.registry` already, under which `sweepers` should reside?

Reply:
Let's open a ticket for that if it's desirable to change